lib/mocha/parameter_matchers/last_positional_hash.rb

```diff
@@ -0,0 +1,26 @@
+require 'mocha/parameter_matchers/base'
+
+module Mocha
+  module ParameterMatchers
+    # Parameter matcher which matches when actual parameter equals expected value.
```

Comment:
Thanks for fixing this! Do you think the name is appropriate? Technically it's not always a last positional hash

Reply:
> Thanks for fixing this!

No worries.

> Do you think the name is appropriate? Technically it's not always a last positional hash.

I assume you mean the name of the class, i.e. `LastPositionalHash`. If so, I see what you mean. Have you got any suggestions for an alternative name? Presumably it could either be the last positional Hash OR the keyword arguments...?